requirements.txt

```diff
@@ -0,0 +1,13 @@
+beautifulsoup4==4.12.2
+certifi==2023.7.22
+charset-normalizer==3.3.1
+Faker==19.12.0
```

Comment:
no pip-tools + requirements.in or poetry + lock file?

Reply:
Thanks for pointing this out. I'll add pip-tools.